docs/resources/resources.md

```diff
@@ -0,0 +1,38 @@
+---
+id: resources
+title:  "Resources"
+---
+
+## Blog Articles
```

Comment:
Could you add a note that these articles presents ideas and code is outdated because dynamic development of ZIO?

Reply:
Good point, added.